File: backend/core/src/modules/project-members/project-members.controller.ts
import { Controller, UseGuards } from '@nestjs/common';
import { ProjectScopedCrudController } from '../../common/access/project-scoped-crud.controller';
import { ProjectMembersService } from './project-members.service';
import { ProjectMember } from '../../entities';
import { JwtAuthGuard } from '../auth/jwt-auth.guard';

@UseGuards(JwtAuthGuard)
@Controller('project-members')
export class ProjectMembersController extends ProjectScopedCrudController<ProjectMember> {
  constructor(service: ProjectMembersService) {
    super(service);
  }
}

File: backend/core/src/main/java/store/pjcloud/cowrite/core/controller/ProjectMembersController.java
package store.pjcloud.cowrite.core.controller;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import store.pjcloud.cowrite.core.common.ProjectScopedCrudController;
import store.pjcloud.cowrite.core.entity.ProjectMember;
import store.pjcloud.cowrite.core.service.ProjectMembersService;

@RestController
@RequestMapping("/project-members")
public class ProjectMembersController extends ProjectScopedCrudController<ProjectMember> {
    public ProjectMembersController(ProjectMembersService service) {
        super(service);
    }
}
